pom.xml

```diff
@@ -1339,5 +1339,27 @@
         </plugins>
       </build>
     </profile>
+    <profile>
+      <id>disableWizard</id>
+      <activation>
+        <property>
+          <name>!enableWizard</name>
+        </property>
+      </activation>
+      <build>
+        <plugins>
+          <plugin>
+            <groupId>org.jenkins-ci.tools</groupId>
+            <artifactId>maven-hpi-plugin</artifactId>
+            <configuration>
+              <systemProperties>
+                <name>hudson.Main.development</name>
+                <value>true</value>
```

Comment:
Yes, it works, but sure, the simpler way would work as well. So I'll move to that. My question now is (applies to both the simple and complicated solution): we will have `hudson.Main.development=true` during JTH tests. I don't see any problem in that, is there?